lib/active_record_shards/model.rb

```diff
@@ -9,13 +9,14 @@ def not_sharded
     end
 
     def is_sharded? # rubocop:disable Style/PredicateName
+      sharded_ivar = defined?(@sharded) ? @sharded : nil
```

Comment:
nope, `defined?` returns `nil` not `false`. But the intention is at least clear like this.